src/inputs/input/input.jsx

```diff
@@ -265,73 +369,50 @@ class Input extends React.PureComponent {
         this.onNumberToggleDownClick = this.onNumberToggleDownClick.bind(this);
         this.onNumberToggleUpClick = this.onNumberToggleUpClick.bind(this);
 
+        this.inputRef = React.createRef();
         this.inputTimer = null;
         this.previousInputValue = '';
     }
 
     componentDidMount() {
-        const { autoFocus, icon, loading } = this.props;
-        const type = this.getType();
-
-        if (isString(icon) || isObject(icon) || loading || type === 'number') {
-            // eslint-disable-next-line react/no-find-dom-node, no-underscore-dangle
-            const inputTop = ReactDOM.findDOMNode(this._input).offsetTop;
-
-            if (inputTop > 0) {
-                this.setState({ inputActionsTopPosition: inputTop });
-            }
-        }
+        const { autoFocus } = this.props;
 
         if (autoFocus) {
             // eslint-disable-next-line react/no-find-dom-node, no-underscore-dangle
-            ReactDOM.findDOMNode(this._input).focus();
+            // ReactDOM.findDOMNode(this.inputRef).focus();
```

Comment:
Commented out on purpose?  Are we going to remove?  Is there a TODO here?

Reply:
Removed.

Reply:
Gonzo!  Resolving.